packages/next/taskfile.js

```diff
@@ -24,6 +27,23 @@ const babelOpts = {
   ]
 }
 
+const babelServerOpts = {
+  presets: [
+    '@babel/preset-typescript',
+    ['@babel/preset-env', {
+      modules: 'commonjs',
+      targets: {
+        node: '8.3'
+      },
+      exclude: ['transform-typeof-symbol']
+    }]
+  ],
+  plugins: [
+    'babel-plugin-dynamic-import-node',
```

Comment:
This looks incorrect, it should not compile `import()`

Reply:
@timneutkens That's only for the server files used by the core, and Node can't handle that.

Reply:
But it wasn't i think

Reply:
@timneutkens I removed `babel-plugin-dynamic-import-node` from everywhere except the `bin` task 👍